frontend/app/services/EventbriteService.scala

```diff
@@ -18,9 +18,25 @@ import utils.StringUtils._
 import scala.concurrent.{ExecutionContext, Future}
 import scala.concurrent.duration._
 
-abstract class EventbriteService(implicit val ec: ExecutionContext, system: ActorSystem) extends WebServiceHelper[EBObject, EBError] {
+case class OrganisationId(value: String) extends AnyVal
+case class EBAccount(apiToken: String, organisationId: OrganisationId)
+
+object EBAccount {
+  val live = EBAccount(
+    Config.eventbriteApiToken,
+    OrganisationId("104062388105")
+  )
+  val masterclass = EBAccount(
+    Config.eventbriteMasterclassesApiToken,
+    OrganisationId("7266571413")
+  )
+}
+abstract class EventbriteService(ebAccount: EBAccount)(
+  implicit val ec: ExecutionContext,
+  system: ActorSystem
+) extends WebServiceHelper[EBObject, EBError] {
 
-  val apiToken: String
+  import ebAccount._
```

Comment:
I decided to make the EBAccount self contained and group the id with the key, rather than making the service classes bigger. it did add a bit of boilerplate though in making it more explicit.